tests/integrations/raftstore/test_stats.rs

```diff
@@ -399,16 +399,20 @@ fn check_query_num_read(
     kind: QueryKind,
     expect: u64,
 ) -> bool {
-    for _ in 0..100 {
-        let hot_peers = cluster.pd_client.get_store_hotspots(store_id).unwrap();
-        let peer_stat = hot_peers.get(&region_id).unwrap();
-        let query_stat = peer_stat.get_query_stats();
-        if QueryStats::get_query_num(query_stat, kind) == expect {
-            return true;
-        }
+    let start = std::time::SystemTime::now();
```

Comment:
Why `SystemTime` is used?

Reply:
used to determine whether to enter an infinite loop

Reply:
Why can't `std::time::Instant` detect it?

Reply:
ok